Tools/CISupport/ews-build/steps.py

```diff
@@ -2582,10 +2582,20 @@ def getQueueStatusFromList(self, change_id, pr_number):
         failed_checks = []
         url = '{}status/{}/'.format(EWS_URL, change_id)
 
-        response = yield TwistedAdditions.request(url, logger=lambda content: self._addToLog('stdio', content))
-        if response and response.status_code // 100 != 2:
-            yield self._addToLog('stdio', f'Accessed {url} with unexpected status code {response.status_code}.\n')
-            return defer.returnValue(False if response.status_code // 100 == 4 else None)
+        max_attempts = 3
+        for attempt in range(1, max_attempts + 1):
```

Comment:
In case of successful response, do we break from the loop?

Reply:
Ah good catch, oops